invenio_records_rest/serializers/loaders/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Loaders."""
+
+from .marshmallow import json_patch_loader, marshmallow_loader
+from ..schemas.json import RecordSchemaJSONV1
+
+json_v1 = marshmallow_loader(RecordSchemaJSONV1)
```

Comment:
Should they actually be here (same goes for ``invenio_records_rest/serializers:json_…``)? I think we originally added them in ``invenio_records_rest/serializers:json_`` because we thought we could reuse them with the need for further schema, but that's no longer the case as far as I'm aware?